corehq/apps/reports/models.py

```diff
@@ -1036,12 +1039,7 @@ def case_exports(self):
 
     @classmethod
     def by_domain(cls, domain):
-        return cache_core.cached_view(cls.get_db(), "groupexport/by_domain",
-            key=domain,
-            reduce=False,
-            include_docs=True,
-            wrapper=cls.wrap,
-        )
+        return hq_group_export_configs_by_domain(domain, cls)
```

Comment:
I think we want to follow the same `quickcache` + `QuickCachedDocumentMixin` pattern I've been talking about. You basically want to do that whenever you remove a view from a `GenerationCache` subclass, because often that caching is actually making a big difference (and shielding our db). Sometimes it's not, but it's hard to tell ahead of time, and would be difficult to pinpoint even if it did cause a problem, so barring further research it's always a good baseline default to preserve view caching (possibly in some other form like I'm suggesting) if possible.